res/sections/23-Les18042017.tex

```diff
@@ -121,6 +121,22 @@ \subsection{Step 9: Prosieguo}
 Il \textit{follow-up} serve a verificare se il management ha intrapreso le azioni
 necessarie a correggere i problemi in tempo.
 
+Dopo aver consegnato il report dell'audit, l'auditor deve condurre
+delle interviste d'uscita con il management per ottenere il suo
+impegno per implementare le raccomandazioni contenute nel report.
+Dopo aver preso nota delle raccomandazioni, il management deve
+indicare le azioni correttive che vuole implementare e una data
+approssimativa per l'implementazione.
+
+È compito dell'auditor -- nei successivi audit --
+controllare che il management abbia onorato
+i propri impegni nel sistemare o rimediare a deficienze trovate negli
+audit precedenti.
+Alcuni problemi sono lasciati non corretti perchè alcuni cambi nella
```

Comment:
Non mi piace la frase:
>Alcuni problemi sono lasciati non corretti perchè alcuni cambi nella
pratica o nel design organizattivo hanno eliminato le condizioni
delle debolezze riscontrate nei controlli precedenti

La cambierei in:
>Alcuni problemi non vengono corretti perchè cambiamenti nella pratica o nel design
organizattivo hanno eliminato le debolezze riscontrate nei controlli precedenti